substra/sdk/config.py

```diff
@@ -12,27 +12,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import copy
+import abc
 import json
 import logging
 import os
 
 logger = logging.getLogger(__name__)
 
 DEFAULT_PATH = os.path.expanduser('~/.substra')
+DEFAULT_TOKENS_PATH = os.path.expanduser('~/.substra-tokens')
 DEFAULT_VERSION = '0.0'
-DEFAULT_URL = 'http://127.0.0.1:8000'
 DEFAULT_PROFILE_NAME = 'default'
```

Comment:
see previous comment about profile_name being required, is this still useful ?

Reply:
I think it is, otherwise when using the CLI it'll always be required to specify a profile name, which could be very annoying.